envsec/internal/jetcloud/project.go

```diff
@@ -40,12 +40,17 @@ func InitProject(ctx context.Context, user *auth.User, wd string) (typeids.Proje
 	repoURL, _ := gitRepoURL(wd)
 	subdir, _ := gitSubdirectory(wd)
 
-	projectID, err := newClient().newProjectID(ctx, user, repoURL, subdir)
+	projectID, err := newClient().newProjectID(ctx, tok, repoURL, subdir)
 	if err != nil {
 		return typeids.NilProjectID, err
 	}
 
-	orgID, err := user.OrgID()
+	claims := tok.IDClaims()
+	if claims == nil {
+		return typeids.NilProjectID, errors.Errorf("token did not contain an org")
+	}
```

Comment:
This code will never get exercised because newProjectID will fail if required claims are not set. 

In fact, any token without claims or missing claims like orgID should not be a valid token and maybe rejected further up the stack?

Reply:
Right now the token from the auth library allows for cases when you have an access token, but no ID token. If we want to always, 100% guarantee the existence of the ID token, I can change the struct so the claims are *not* a pointer (and thus, can't be null)

Should we go that route?